styles/Home.module.scss

```diff
@@ -18,6 +18,50 @@
   flex-direction: column;
   justify-content: center;
   align-items: center;
+  position: relative;
+  z-index: 10;
+}
+
+.hero{
+  position: absolute;
+  top: 0;
+  left: 0;
+  height: 100vh;
+  width: 100vw;
+  text-transform: uppercase;
+  font-size: 20vw;
+  font-weight: bold;
+  text-align: center;
+  line-height: 18vw;
+  display: grid;
+  place-content: center;
+  color: colors.$white;
+  opacity: 0.01;
+  user-select: none;
+  transition: opacity 350ms;
+  @media screen and (max-width: 800px){
+    opacity: 0;
+  }
+}
+
+.socials{
+  display: grid;
+  color: colors.$disabled;
+  font-size: 1.3rem;
+  text-align: center;
+  & p{
```

Comment:
space between p and { here

Reply:
updated all the tags to have spaces now

Reply:
also, I've been writing alot of css without the space there and yet in every snippet I see, there exists a space. Any reason or resource you can find that says it's necessary?

Reply:
Syntactically, no, but stylistically definitely. 
https://courses.cs.washington.edu/courses/cse154/17au/styleguide/html-css/spacing-indentation-css.html#spacing-css
At the bottom, it is explained. its a CSS convention that carries too any inheriting syntax.